README.md

```diff
@@ -693,17 +742,54 @@ Define the target field name used to retrieve the label of the referenced elemen
                 .targetField(nga.field('title')) // Select a label Field
         ]);
         
-* `singleApiCall(function(entityIds) {}`
+* `singleApiCall(function(entityIds) {})`
+
 Define a function that returns parameters for filtering API calls. You can use it if you API support filter for multiple values.
 
-		// Will call /posts?post_id[]=1&post_id[]=2&post_id%[]=5...
+		// Will call /posts?_filters=id1,id2,.._
```

Comment:
Ew, then it was bad from the start on. Can't we do something BC to allow a default url as `/posts?post_id[]=1&post_id[]=2&...` without interceptor?